src/applications/disability-benefits/view-payments/actions/index.js

```diff
@@ -19,8 +22,15 @@ export const getAllPayments = () => async dispatch => {
     // TODO: fire off analytics event when endpoint is wired up.
     //   const errCode = res.errors[0].code;
     //   isServerError(errCode) ? recordEvent({}) : recordEvent({})
+    recordEvent({
+      event: `disability-view-dependents-load-failed`,
+      'error-key': `${response.errors[0].status}_description_of_error`,
```

Comment:
I think this key is looking for an actual descriptor as opposed to the string `description_of_error`. Even though we haven't hooked redux up to the backend yet, we can probably still add those in and keep the messages general, similar to what's done in rated disabilities.

Reply:
on rated disabilities, we use the error code concatenated with `_internal_error` unless Im looking in the wrong place?

Reply:
right, so here we'd probably want to do the same thing, using a word more pertinent to the error than `description`. For example `503_server_error` or `404_client_error`

Reply:
werd, just added this.